packages/webapp/src/routes/land/add-landowner-individual.js

```diff
@@ -0,0 +1,58 @@
+import constants from '../../utils/constants.js'
+import {
+  getLegalAgreementDocumentType,
+  validateFirstLastName
+} from '../../utils/helpers.js'
+import isEmpty from 'lodash/isEmpty.js'
+
+const handlers = {
+  get: async (request, h) => {
```

Comment:
IN progress status missing 
processRegistrationTask(request, {
      taskTitle: 'Legal information',
      title: 'Add legal agreement details'
    }, {
      inProgressUrl: =
    })

Reply:
done